json-endpoints/jetpack/class.jetpack-json-api-themes-delete-endpoint.php

```diff
@@ -24,7 +24,7 @@ protected function delete() {
 			$result = delete_theme( $theme );
 
 			if ( is_wp_error( $result ) ) {
-				$error = $this->log[ $theme ]['error'] = $result->get_error_messages;
+				$error = $this->log[ $theme ]['error'] = $result->get_error_messages();
```

Comment:
I wondered if we actually wanted to do `$result->get_error-message()` here to get a singular error, but since we [don't return this error, and instead just check that an error is present](https://github.com/Automattic/jetpack/blob/a1f18db33a409edc5c29dc573dbed85feb4d21a3/json-endpoints/jetpack/class.jetpack-json-api-themes-delete-endpoint.php#L35), this should be OK.